src/modules/Authentictation/Components/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,7 +1,148 @@
-import React from 'react'
+import {
+  Box,
+  Button,
+  FormHelperText,
+  TextField,
+  Typography,
+} from "@mui/material";
+import { red } from "@mui/material/colors";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import { getValidationRules } from "../../../../services/Validations";
+import { AUTH_URLS, axiosInstance } from "../../../../services/urls";
+import CircularProgress from "@mui/material/CircularProgress";
+import { toast } from "react-toastify";
+import { User } from "../Registeration/Registeration";
+
+const ForgetPassword = () => {
+  const navigate = useNavigate();
+
+  const {
+    formState: { errors, isSubmitting },
+    register,
+    handleSubmit,
+    watch,
+  } = useForm<User>({
+    mode : "onChange"
+  });
+
+  const validationRules = getValidationRules(watch);
+
+  const onSubmitHandler = async (data: User) => {
+    await axiosInstance
+      .post(AUTH_URLS.forgetPassword, data)
+      .then((response) => {
+        console.log(response?.data?.message);
+        toast.success(
+          response?.data?.message || "please check your email for OTP"
+        );
+        navigate("/reset-password", { state: data.email });
+      })
+      .catch((error) => {
+        console.log(error);
+        toast.error(
+          error?.response?.data?.message ||
+            "something went wrong please try again"
+        );
+      });
+  };
 
-export default function ForgetPassword() {
   return (
-    <div>ForgetPassword</div>
-  )
-}
+    <>
+      <Box
+        component="div"
+        sx={{
+          ml: { md: "7rem", xs: "2rem" },
+          mt: { md: "3.9rem", xs: "2rem" },
+        }}>
+        <Typography component="h1" sx={{ fontSize: "1.87rem" }}>
+          Forgot password
+        </Typography>
+
+        <Box component="div" mt="22px">
+          <Typography component="p">
+            If you already have an account register
+            <Typography component="span" display={"block"}>
+              You can{" "}
+              <Box
+                ml={1}
+                sx={{ color: red[600] }}
+                fontWeight={600}
+                component={"span"}>
```

Comment:
We can give  any MUI component an html element or a custom component to be rendered as the actual html structure. We can here for example give the Link component to the box instead of nesting it inside the box.